CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Conributing
+
+## What not to commit to the repo
+
+There are a handful of things we do not commit to the repository:
+- Information about particular vendors or procurement information
+- Anything related to PII whether it is from user interviews or feedback
+- User research related scheduling or documentation
```

Comment:
I would like to add user research synthesis to our repo at some point to demonstrate our understanding. This doesn't need to be a blocker, but something to consider later on.